lib/fetch/index.js

```diff
@@ -850,13 +858,13 @@ async function schemeFetch (fetchParams) {
 
       // 4. Return a response whose status message is `OK`,
       //    header list is « (`Content-Type`, mimeType) »,
-      //    and body is dataURLStruct’s body.
+      //    and body is dataURLStruct’s body as a body.
       return makeResponse({
         statusText: 'OK',
         headersList: [
           ['content-type', mimeType]
         ],
-        body: extractBody(dataURLStruct.body)[0]
+        body: safelyExtractBody(dataURLStruct.body)[0]
```

Comment:
Why safely?

Reply:
https://fetch.spec.whatwg.org/#byte-sequence-as-a-body

"To get a [byte sequence](https://infra.spec.whatwg.org/#byte-sequence) bytes as a body, return the [body](https://fetch.spec.whatwg.org/#body-with-type-body) of the result of [safely extracting](https://fetch.spec.whatwg.org/#bodyinit-safely-extract) bytes."